javascript/javascript2/week1/hyf_bay/hyfBayHelper.js

```diff
@@ -0,0 +1,72 @@
+/* DONT MODIFY ANY OF THIS CODE!!!*/
+
+window.getAvailableProducts = function() {
```

Comment:
we are creating nested functions. when you have time try rather than this  in object > method way
https://codepen.io/naresh699/pen/yoZZJe?editors=1010